doc/source/dev/roadmap-detailed.rst

```diff
@@ -431,10 +424,7 @@ The following improvements will help SciPy better serve this role.
   - Handle censored data (e.g. merge `gh-13699 <https://github.com/scipy/scipy/pull/13699>`__)
 
 - Implement additional widely used continuous and discrete probability
-  distributions:
-
-  - multivariate t distribution
-  - mixture distributions
+  distributions, e.g. mixture distributions.
```

Comment:
Not sure about mixture because we ruled out copulas and consequently it's now in statsmodels.